### PR TITLE
Improvement: Decouple kit

### PR DESCRIPTION
**Pull request by snova-luiss**

### CODE REVIEW

File: search_assistant/src/search_assistant.py [hunk 1]
```diff
@@ -32,15 +31,14 @@
...
-from utils.visual.env_utils import get_wandb_key
 
 CONFIG_PATH = os.path.join(kit_dir, 'config.yaml')
 PERSIST_DIRECTORY = os.path.join(kit_dir, 'data/my-vector-db')
 
 load_dotenv(os.path.join(repo_dir, '.env'))
 
 # Handle the WANDB_API_KEY resolution before importing weave
-wandb_api_key = get_wandb_key()
+wandb_api_key = None  # get_wandb_key()
```
Comment (snova-jorgep) [inline search_assistant/src/search_assistant.py:41]: why is this being initialized with None now?

Comment (snova-luiss) [reply search_assistant/src/search_assistant.py]: Fixed!

**Review by snova-jorgep — APPROVED** [review]

File: search_assistant/src/search_assistant.py [hunk 2]
```diff
@@ -38,7 +38,7 @@
 load_dotenv(os.path.join(repo_dir, '.env'))
 
 # Handle the WANDB_API_KEY resolution before importing weave
-wandb_api_key = None  # get_wandb_key()
+wandb_api_key = get_wandb_key()
```
Comment (snova-jorgep) [inline search_assistant/src/search_assistant.py:41]: get_wandb_key is not imported

**Review by snova-jorgep — CHANGES_REQUESTED** [review]

**Review by snova-jorgep — APPROVED** [review]
LGTM Thanks!

**Pull request merged**